README.md

```diff
@@ -1,3 +1,5 @@
 What a nice page!
 
 ![?](https://octodex.github.com/images/homercat.png)
+
+Please improve!
```

Comment:
I think it looks great and does not need to be improved. But take a look at this ![GitHub Logo](https://octodex.github.com/images/gangnamtocat.png)